netlify.toml

```diff
@@ -1,3 +1,15 @@
 [build]
   publish = "_site/"
   command = "git submodule update --init --recursive && JEKYLL_ENV=production ./bin/build"
+
+[[redirects]]
+  from = "/cfp"
+  to = "https://www.papercall.io/rubyconfth2022"
+  
+[[redirects]]
+  from = "/tickets"
+  to = "https://www.eventpop.me/e/13417/rubyconfth-2022"
```

Comment:
I'm getting a 404 on this, but I guess it's not live yet, is it?

Reply:
Correct